src/service/ui/mousepad.js

```diff
@@ -100,6 +100,8 @@ export const InputDialog = GObject.registerClass({
 
     _init(params) {
         super._init(Object.assign({
+            application: Gio.Application.get_default(),
+            title: _('Remote Input'),
```

Comment:
This is set by defining the `headerbar.title`, which is the title of a GtkDialog in every way that matters.

In the application dash and overview:

![image](https://github.com/user-attachments/assets/669328d0-a5b5-4bb3-9547-5d9d9612b629) ![image](https://github.com/user-attachments/assets/16ce7f71-8937-41f8-82df-12785e5c77ca)

In the Gtk Inspector:

![image](https://github.com/user-attachments/assets/904bff29-d645-4113-8f36-b04cbacc623b)

![image](https://github.com/user-attachments/assets/953945fd-467b-49b4-84ff-dc3dc116d67e)

What issue are you trying to solve, by also passing a `title` parameter to the constructor?